Makefile

```diff
@@ -95,6 +95,10 @@ rpm-release: srpm-release
 	if test ! -d BUILD/RPM; then mkdir -p BUILD/RPM; fi
 	mock --resultdir BUILD/RPM -D "commit $(RELEASE_COMMIT)" --rebuild BUILD/SRPM/avocado-plugins-vt-$(VERSION)-*.src.rpm
 
+requirements:
+	- pip install "pip>=6.0.1"
```

Comment:
Requirement files from `avocado-vt` are compatible with older pip, so this is not needed (it's required by `avocado` requirement files as we use the versions feature added in pip-6.0.1)